File: InStock/InStock/ClientApp/src/app/models/inventory.ts
export class Inventory {
    sku?: number;
    available: number;
    quantity: number;
    shopId: number;
    itemId: number;
    items:{
        id:number;
        name:string;
    };
    shops:{
        shopId:number;
        name:string;
    };
}



 
//     "shops": {
//       "shopId": 1,
//       "name": "Kmart",
//       "contactNumber": null,
//       "address": null,
//       "lat": 0,
//       "long": 0
//     }
//   },

File: InStock/_DAL/DALModels/Inventory.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace _DAL.DALModels
{
    public class Inventory
    {
        [Key]
        public int SKU { get;   set; }
        [Required]
        public int Available { get; set; }
        [Required]
        public int Quantity { get;   set; }
        public int ItemId { get; set; }
        public int ShopId { get; set; }
        public Item Item { get; set; }
        public Shop Shop { get; set; }

    }
}
